superchain/superchain.go

```diff
@@ -60,6 +60,31 @@ type ChainConfig struct {
 	// Chain is a simple string to identify the chain, within its superchain context.
 	// This matches the resource filename, it is not encoded in the config file itself.
 	Chain string `yaml:"-"`
+
+	// Hardfork Configuration Overrides
+	CanyonTime  *uint64 `yaml:"canyon_time,omitempty"`
+	DeltaTime   *uint64 `yaml:"delta_time,omitempty"`
+	EcotoneTime *uint64 `yaml:"ecotone_time,omitempty"`
+	FjordTime   *uint64 `yaml:"fjord_time,omitempty"`
+}
+
+// replaceMissingOverridesWithDefaults overwrites each unspecified hardfork activation time override
+// with the superchain default.
+func (c *ChainConfig) replaceMissingOverridesWithDefaults(s Superchain) {
+
```

Comment:
nit: linter probably won't accept fist and last empty lines in function. The linter run currently errors.

Reply:
Be nice if my editor just fixed this for me automatically. Does yours?

Reply:
yes, always applies gofmt on save :)

Reply:
I don't think the linter was actually upset by this. If we want it to be, it may require some configuration change.